src/encryption/encryption.ts

```diff
@@ -266,33 +223,26 @@ export function encryptBinaryWithKeyUsingArtefacts(
   encrypted: string | null;
   artifacts?: any;
 } {
-  return encryptWithKeyUsingArtefacts(key, data, strategy, iv, 'raw');
+  return encryptWithKeyUsingArtefacts({ key, data: binaryToBytes(data), strategy, iv });
```

Comment:
`decodeByteString` here - or just use the existing `stringAsBinaryBuffer` method that exists in the codebase.